browser/Makefile.am

```diff
@@ -1046,6 +1046,7 @@ pot:
         src/slideshow/engine/animationnodes/NodeContext.ts \
         src/slideshow/engine/animationnodes/PropertyAnimationNode.ts \
         src/slideshow/engine/AnimatedElement.ts \
+        src/slideshow/PresenterConsole.js \
```

Comment:
Good catch. Do you have a bash oneliner to try to find files which use `_(...)` and are not listed in the pot target? If not, perhaps it would be good to have one, maybe this is not the only missing file.

Reply:
You are right. There is more to do, I just sent a quick fix last night to react to a bug report. I compared the list of files that should be here to the list of files that are actually here, and I detected issues. I'll clean up the list in a follow-up PR. For example many files in this list do not contain localizable strings, so they are there unnecessarily. I found some more files to add, too.